README.md

```diff
@@ -66,3 +66,45 @@ This middleware should be declared *after* your other routes but *before* your e
 
 `translate` can be the HOF i18n translate function
 
+## Deep translate
+
+deepTranslate middleware supports nested conditional translations in order to show different content in different scenarios. The middleware adds a `translate` function to `req` which is used in various points throughout the architecture.  This middleware must be applied before any other middleware which rely on the `req.translate` function. Also when initializing the form wizard, or template mixins, if a `translate` function is provided, this will be used rather than the deepTranslate middleware.
+
```

Comment:
That would be reverse functionality though. This is opt in, if you didn't want to use it you wouldn't apply the middleware